src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const item of items) {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
+}
+
+tree.onclick = (e) => {
+  if (e.target.tagName !== 'SPAN') {
+    return;
+  }
+
+  const childrenContainer = e.target.parentNode.querySelector('ul');
+
+  childrenContainer.hidden = !childrenContainer.hidden;
```

Comment:
The line `childrenContainer.hidden = !childrenContainer.hidden;` assumes that `childrenContainer` is not null. If the list item does not have a child `ul`, `childrenContainer` will be null, and this line will throw an error. Consider adding a check to ensure `childrenContainer` is not null before attempting to toggle its hidden property.